bootstrap/bootstrap.go

```diff
@@ -160,6 +160,22 @@ func (b *Bootstrap) Run(ctx context.Context) (exitCode int) {
 	if phaseErr == nil && includePhase(`command`) {
 		var commandErr error
 		phaseErr, commandErr = b.CommandPhase(ctx)
+		/*
+			Five possible states at this point:
+
+			Pre-command failed
+			Pre-command succeeded, command failed, post-command succeeded
```

Comment:
This change only affects this case, if the post-command succeeds there is no `phaseError` to report.